data_structures/src/chain.rs

```diff
@@ -573,7 +572,17 @@ pub struct SuperBlock {
 
 impl Hashable for SuperBlock {
     fn hash(&self) -> Hash {
-        calculate_sha256(&self.to_pb_bytes().unwrap()).into()
+        let superblock_data = [
+            &self.ars_length.to_be_bytes()[..],
+            self.ars_root.as_ref(),
+            self.data_request_root.as_ref(),
+            &self.index.to_be_bytes()[..],
+            self.last_block.as_ref(),
+            self.last_block_in_previous_superblock.as_ref(),
+            self.tally_root.as_ref(),
+        ]
+        .concat();
```

Comment:
Maybe move this to a method of `SuperBlock`, so it is decoupled from the `Hashable` impl. And add some documentation there, something along the lines of "serialize the superblock by converting all the fields to bytes and concatenating them". Also note that this serialization method is super simple, but it only works if all the fields have a known fixed size.

```suggestion
        let superblock_data = self.serialize_as_bytes();
```

Reply:
So add more tests, and also explain why aren't we using protocol buffers (because this way it's easier to verify superblocks in the EVM)